examples/el_farol/Readme.md

```diff
@@ -0,0 +1,26 @@
+# El Farol
+
+This folder contains an implementation of El Farol restaurant model. Agents (simulated restaurant customers) decide whether to go to the restaurant or not based on their memory and reward from previous trials. Implications from the model have been used to explain how individual decision-making affects overall performance and fluctuation.
+
+Besides, it demonstrates how to deploy a cognitive model (Instance-Based Learning) under the mesa environment. IBL model reflects the recency and frequency effect in decision-making with memory. Agent actively learns from the environment and updates their preference(blending value) for each decision. IBL model could be used as a substitute for an agent whose decision-making is more realistic and closer to human decision-making. 
+IBL model deployed from the PyIBL package was used to replicate the research paper [1]. 
```

Comment:
Typo: "was" -> "is".

Reply:
I think this sentence should be replaced with:

This example has 2 versions of the model: the pure Mesa version without IBL (named as `ElFarolBar`), and the version with IBL that integrates with PyIBL (named as `ElFarolBarIBLT`). The latter version replicates the result of Kumar 2016.